hack/install.sh

```diff
@@ -11,7 +11,9 @@
 # shellcheck disable=SC1091,SC1090
 source "$(dirname "${BASH_SOURCE[0]}")/lib/__sources__.bash"
 
-set -Eeuo pipefail
+set -Eeo pipefail
```

Comment:
Why drop the `u`?

Reply:
Added back the `u` option.

`u` option does not allow empty value, so I dropped it to accept `install_prev="$1"` with empty. But never mind, as I could use env value by using `make install-previous`.